staubli_resources/urdf/common_colours.xacro

```diff
@@ -2,9 +2,15 @@
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
 
   <!-- RAL 1028 -->
-  <xacro:property name="colour_staubli_yellow" value="1.0 0.608 0 1" />
+  <xacro:property name="melon_yellow" value="1.0 0.608 0.0 1.0" />
```

Comment:
Also: I've been changing the way in which we define these colours: instead of doing the `int`->`float` conversion externally, I let xacro do it for us (see [fanuc_resources](https://github.com/ros-industrial/fanuc/blob/a859053f5b601012d279e74a7d769c8b428640fc/fanuc_resources/urdf/common_colours.xacro#L12-L25)) fi).

This saves us work (we don't have to do the conversion), and it makes it easier to figure out which particular RGB values were used for the RAL colour (seeing as not all sites give the same values for the same RAL colour).

Ideally we'd also include a comment stating which particular site was used to source the RGB values (like [abb_resources/common_colours.xacro](https://github.com/ros-industrial/abb_experimental/blob/ffb631009d044d7a38e7de485e3e9e2e65b924c7/abb_resources/urdf/common_colours.xacro#L3-L4). You don't have to use that site, it is just an example).